### PR TITLE
Improvisations Suggested

### PR DESCRIPTION
**Pull request by Ved-programmer**
# NOTE
    
Removed:

```
# record the start datetime
self.start_dt = datetime.now()
```
It's not required in the start method, since it's already done in the initialisation

### CODE REVIEW

**Review by Ved-programmer — COMMENTED** [review]
# Improvisations Suggested

* Removed repetetive code
* Removed redundant/unnecessary comments
* Added blank lines in certain places to differentiate between separate pieces of code and for easability of reading

**Comment by crypto-navdeep** [comment]
Hi Ved! I Just Checked it now.

Thanks for your contribution, I'm currently busy now, I'll check it later.
Nevertheless, Thanks a lot for your time and effort for reviewing the code, I'll check and merge PR as soon a possible.

Regards,
Navdeep.

File: keylogger.pyw [hunk 1]
```diff
@@ -16,83 +16,92 @@ class Keylogger:
...
-        # set the thread as daemon (dies when main thread die)
-        timer.daemon = True
-        # start the timer
+        
+        timer.daemon = True # set the thread as daemon (dies when main thread die)
         timer.start()
 
     def start(self):
-        # record the start datetime
-        self.start_dt = datetime.now()
```
Comment (Ved-programmer) [inline keylogger.pyw:86]: self.start_dt has already been defined in the initialisation

**Comment by Ved-programmer** [comment]
> Hi Ved! I Just Checked it now.
> 
> Thanks for your contribution, I'm currently busy now, I'll check it later.
> Nevertheless, Thanks a lot for your time and effort for reviewing the code, I'll check and merge PR as soon a possible.
> 
> Regards,
> Navdeep.

Ok

**Review by crypto-navdeep — APPROVED** [review]
Great :)

**Pull request merged**